dynamodb/helper.js

```diff
@@ -0,0 +1,10 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+  region: process.env.AWS_DYNAMODB_REGION ? process.env.AWS_DYNAMODB_REGION : 'local'
```

Comment:
Would it be better to default the region to `eu-west-1` rather than `local` to prevent errors if the user deploys without setting the region specifically?

My understanding on dynamodb-local is that region is inconsequential?